pubnub-chat-impl/src/commonMain/kotlin/com/pubnub/chat/internal/UserImpl.kt

```diff
@@ -98,15 +98,21 @@ data class UserImpl(
         val effectiveFilter: String = filter?.let { "$internalModerationFilter && ($filter)" } ?: internalModerationFilter
 
         return chat.pubNub.getMemberships(
-            uuid = id,
+            userId = id,
             limit = limit,
             page = page,
             filter = effectiveFilter,
             sort = sort,
-            includeCount = true,
-            includeCustom = true,
-            includeType = true,
-            includeChannelDetails = getChannelDetailsType(true)
+            include = MembershipInclude(
+                includeCustom = true,
+                includeStatus = false,
+                includeType = false,
+                includeTotalCount = true,
+                includeChannel = true,
+                includeChannelCustom = true,
+                includeChannelType = true,
+                includeChannelStatus = false
```

Comment:
we should probably set all includes to true everywhere, otherwise there's no way to get e.g. status and type for memberships, since the user can't override it